themes/foulab-2018/static/css/default/app.css

```diff
@@ -1,23 +1,69 @@
-/* foulab.org - default style */
-
-/* Header of the page */
-.header
-{
-    position: relative;
-    width: 880px;
-    height: 168px;
-    float: left;
-    background-color: #000000;
-    background-image: url(../../img/header.png);
-    border: solid #22180b;
-    border-width: 0px 0px 0px 0px;
-    text-align: right;
-    padding-top: 0px;
-    padding-right: 15px;
+/* Main Layout */
+
+#main {
+  display: flex;
+  flex-direction: column;
+  max-width: 890px;
+  margin: 0 auto;
+  box-shadow: 0px 0px 100px 25px rgba(0, 0, 0, .80);
+}
+
+#header {
+  background-color: black;
+  color: white;
+  position: relative;
+  height: 168px;
+  background-image: url(../../img/header.png);
+  border: solid #22180b;
+  border-width: 0 1px;
+  text-align: right;
+  padding-top: 0px;
+}
+
+/* Wraps both the content (left) and the nav (right) */
+#content-wrapper {
+  flex: auto;
+  display: flex;
+  justify-content: space-between;
+  border-left: 1px solid #22180b;
+}
+
+/* Left part of the page */
+#content {
+  background-color: #ffffff;
+  color: #000000;
+  font-size: 10pt;
+  padding: 10px;
+  flex-basis: 0;
+  flex-grow: 999;
+}
+
+#nav {
+  width: 150px;
+  background-color: grey;
```

Comment:
This new background colour makes the nav fail even more on the WCAG contrast guidelines for text
[Current: `4.6:1`](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=757575)
[The Pr: `3.94:1`](https://webaim.org/resources/contrastchecker/)